contracts/colonyNetwork/ColonyNetwork.sol

```diff
@@ -187,11 +187,6 @@ contract ColonyNetwork is BasicMetaTransaction, ColonyNetworkStorage, Multicall
     return changed;
   }
 
-  /// @notice @deprecated
-  function deprecateSkill(uint256 _skillId) public stoppable {
```

Comment:
Why is this happening here, when presumably it should be happening in my `maint/deprecation-removal` branch?

Reply:
I was looking for opportunities to improve coverage, and removing a function which was deprecated ages ago and is no longer testable seemed like a good place to start. I can move it into the other branch, but this branch wont be mergable until that one is.